config/locales/ja.yml

```diff
@@ -448,7 +448,7 @@ ja:
     expires_in_prompt: 無期限
     generate: 作成
     max_uses:
-      one: 1
+      one: "1"
```

Comment:
`"1"` -> `'1'`

double quote to single quote.

Reply:
ahhh I had a single quote but I thought the double quote would be better for consistency. my b. orz